app/import_candidates.py

```diff
@@ -67,7 +67,7 @@ def getStateAbbr(s):
     print 'key ' + s + ' not found!'
     return None
 
-dbpath = '/root/CongressionalGuide/db/db.sqlite3'
+dbpath = '/root/CongressionalGuide/app/db'
```

Comment:
Do you want to change this dbpath before I merge? Otherwise it won't work I think.